app/models/concerns/custom_attribute_definition/broadcastable.rb

```diff
@@ -0,0 +1,14 @@
+module CustomAttributeDefinition::Broadcastable
+  extend ActiveSupport::Concern
+  included do
+    after_create_commit {
+      broadcast_append_later_to "custom_attributes_definitions_#{self.account.id}", target: 'custom_attributes_definitions', partial: 'accounts/settings/custom_attributes_definitions/custom_attribute_definition', locals: {custom_attributes_definition: self}
```

Comment:
Aqui tava pensando de padronizarmos talvez algo como:

```suggestion
      broadcast_append_later_to [self.account, self], target: 'custom_attributes_definitions', partial: 'accounts/settings/custom_attributes_definitions/custom_attribute_definition', locals: {custom_attributes_definition: self}
```

Reply:
Testei dessa forma e não funcionou, pq o canal que irá escutar esse broadcast, será um canal global, e precisamos que esse broadcast faça referencia a esse canal global. Pelo que eu entendi, quando colocamos o self, ele varia o valor dependendo da ação em que ele irá ser executado. Para simplificar e deixar padronizado, eu optei por deixa-lo dessa forma [account, :custom_attribute_definition], agora irá sempre se comunicar com o canal que tenha o mesmo account e q tenha o :custom_attribute_definition, o :custom_attribute_definition por ser global, n terá o problema q estava ocorrendo quando eu testei com o self. FIcou assim: 
```ruby
module CustomAttributeDefinition::Broadcastable
  extend ActiveSupport::Concern
  included do
    after_create_commit do
      broadcast_append_later_to [account, :custom_attribute_definition],
                                target: 'custom_attributes_definitions', partial: 'accounts/settings/custom_attributes_definitions/custom_attribute_definition', locals: { custom_attributes_definition: self }
    end
    after_update_commit do
      broadcast_replace_later_to [account, :custom_attribute_definition], target: self,
                                                                          partial: 'accounts/settings/custom_attributes_definitions/custom_attribute_definition', locals: { custom_attributes_definition: self }
    end
    after_destroy_commit do
      broadcast_remove_to [account, :custom_attribute_definition], target: self
    end
  end
end
``` 
E no index ficou assim: 
```ruby
<%= turbo_stream_from [current_user.account, :custom_attribute_definition] %>
```